### PR TITLE
feat: save user images to user folder and migrate to new file storage api (#36)

### PR DESCRIPTION
**Pull request by vladisavvv**

### CODE REVIEW

File: aidial_adapter_openai/app.py [hunk 1]
```diff
@@ -75,7 +74,12 @@ async def chat_completion(deployment_id: str, request: Request):
 
     if is_text_to_image_deployment(deployment_id):
         return await text_to_image_chat_completion(
-            data, upstream_endpoint, api_key, is_stream, file_storage
+            data,
+            upstream_endpoint,
+            api_key,
+            is_stream,
+            file_storage,
+            request.headers["authorization"],
```
Comment (adubovik) [inline aidial_adapter_openai/app.py]: You need to handle the case when the header is unset.

**Comment by vladisavvv** [comment]
/deploy-review
>Deployment status: success

**Comment by vladisavvv** [comment]
/deploy-review
>Deployment status: success

File: aidial_adapter_openai/app.py (hunk 1 above)
Comment (vladisavvv) [reply aidial_adapter_openai/app.py]: fixed

**Review by adubovik — APPROVED** [review]

**Pull request merged**